test-agents/copilotstudio-webchat/src/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from 'path'
+
+import { loadCopilotStudioConnectionSettingsFromEnv } from '@microsoft/agents-copilotstudio-client'
+import pkg from '@microsoft/agents-copilotstudio-client/package.json' with { type: 'json' }
+import express from 'express'
+
+import { dependency } from './util.js'
+
+const PORT = process.env.PORT || 3978
+const settings = loadCopilotStudioConnectionSettingsFromEnv()
+const settingsString = JSON.stringify(settings, null, 2)
+
+const app = express()
+
+// Serves all static assets from the "public" directory to support the website UI.
+app.use(express.static(path.resolve(import.meta.dirname, '../public')))
+
+// Make the Copilot Studio Client browser library available from node_modules.
+app.use(dependency('@microsoft/agents-copilotstudio-client/browser'))
+
+// Serves the Copilot Studio Client settings to the browser.
+// CAUTION: This endpoint is intended to serve non-sensitive information.
+//          In a real production environment, if you require to serve sensitive information,
+//          always use a secure configuration management to avoid exposing it directly.
+app.get('/agentSettings', (_, res) => {
```

Comment:
I would hardcode the values in a client js file, to follow MSAL node samples. eg: https://github.com/Azure-Samples/ms-identity-docs-code-javascript/blob/3cd6ef52b662e51b755000ea9bccd979a85aaac7/vanillajs-spa/App/public/index.html#L114